operatorcourier/validate.py

```diff
@@ -237,3 +248,179 @@ def _type_validation(self, bundleData, typeName, validator, required):
             valid = False
 
         return valid
+
+    def _ui_validation_ocp(self, bundleData):
+        valid = True
+        logger.info("Validating cluster service versions for OCP OperatorHub UI.")
```

Comment:
is this intended to just do nothing right now?

Reply:
@kevinrizza yeah is that ok? Intending to add / refactor for common code in a future PR. Happy to remove this for now if you think that's better.

Reply:
@galletti94 I think if it does nothing right now we should just omit it. We can come back later and create it when we implement it. Otherwise it could give a false positive.